python/python/lancedb/remote/table.py

```
@@ -554,6 +557,9 @@ def drop_columns(self, columns: Iterable[str]):
     def drop_index(self, index_name: str):
         return LOOP.run(self._table.drop_index(index_name))
 
+    def wait_for_index(self, index_names: Iterable[str], timeout: timedelta = timedelta(seconds=300)):
```

Ruff (0.8.2)

Line too long (102 > 88)

(E501)